src/crimson/os/seastore/transaction_manager.h

```diff
@@ -294,13 +294,14 @@ class TransactionManager : public SegmentCleaner::ExtentCallbackInterface {
   alloc_extent_ret<T> alloc_extent(
     Transaction &t,
     laddr_t laddr_hint,
-    extent_len_t len) {
+    extent_len_t len,
+    placement_hint_t hint = placement_hint_t::HOT) {
```

Comment:
Instead of passing a hint argument here, just set placement_hint in the else branch below based on the configured value you added.